go/libraries/doltcore/sqle/tables.go

```diff
@@ -598,8 +616,142 @@ func (t *WritableDoltTable) getFullTextEditor(ctx *sql.Context) (fulltext.TableE
 			RowCount:    rowCountTable.(fulltext.EditableTable),
 		})
 	}
+	return configTable, sets, nil
+}
 
-	return fulltext.CreateEditor(ctx, t, configTable, sets...)
+// tableSetsForRewrite returns the fulltext.TableSet for each Full-Text index in the table, truncated and modified
+// for a table rewrite operation. Returns the root given with all full-set pseudo tables updated with their new
```

Comment:
I think you meant Full-Text instead of Full-Set? You're using Full-Set in multiple places here lol

Reply:
good catch